### PR TITLE
Update Starbucks

### PR DESCRIPTION
**Pull request by AnalogCyan**

### CODE REVIEW

File: _data/food.yml [hunk 1]
```diff
@@ -39,8 +39,10 @@ websites:
   - name: Starbucks
     url: https://www.starbucks.com/
     img: starbucks.png
-    twitter: Starbucks
-    facebook: Starbucks
+    tfa:
+      - sms
+      - phone
+    doc: https://customerservice.starbucks.com/app/answers/detail/a_id/6180/kw/two%20factor%20authentication
```
Comment (Carlgo11) [inline _data/food.yml]:
```suggestion
    doc: https://customerservice.starbucks.com/app/answers/detail/a_id/6180
```

File: _data/food.yml [hunk 2]
```diff
@@ -39,8 +39,10 @@ websites:
   - name: Starbucks
     url: https://www.starbucks.com/
     img: starbucks.png
-    twitter: Starbucks
-    facebook: Starbucks
+    tfa:
+      - sms
+      - phone
```
Comment (Carlgo11) [inline _data/food.yml]: Are you sure they do phone calls?

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/food.yml (hunk 2 above)
Comment (AnalogCyan) [reply _data/food.yml]: It appeared to be an option but I just checked and it's only an option during setup, all future codes are sent over sms.

Comment (Carlgo11) [reply _data/food.yml]:
```suggestion
```

**Review by Carlgo11 — APPROVED** [review]

**Review by kmpoppe — APPROVED** [review]

**Comment by phallobst** [comment]
Why was the "phone" option removed here? This page says otherwise:
https://customerservice.starbucks.com/app/answers/detail/a_id/6186
Is it not up-to-date anymore?

**Pull request merged**